test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/ProgressSample.cs

```diff
@@ -7,167 +7,179 @@ namespace Tizen.NUI.Samples
-            progressBar1_2.MaxValue = 100;
-            progressBar1_2.MinValue = 0;
-            progressBar1_2.CurrentValue = 15;
-            progressBar1_2.Style.Track.BackgroundColor = Color.Green;
-            progressBar1_2.Style.Progress.BackgroundColor = Color.Black;
-
-            root.Add(progressBar1_2);
-
-            ProgressStyle attr = new ProgressStyle
+            parent = new View()
             {
-                Track = new ImageViewStyle
-                {
-                    BackgroundColor = new Selector<Color>
-                    {
-                        All = Color.Cyan,
-                    }
-                },
-                Progress = new ImageViewStyle
-                {
-                    BackgroundColor = new Selector<Color>
-                    {
-                        All = Color.Black,
-                    }
-                },
-
+                Position = new Position(430, 200),
+                Size = new Size(1000, 730)
+            };
+            parent.Layout = new LinearLayout()
+            {
+                LinearOrientation = LinearLayout.Orientation.Vertical,
+                LinearAlignment = LinearLayout.Alignment.CenterHorizontal
             };
 
-            progressBar2_1 = new Progress(attr);
-            progressBar2_1.Position2D = new Position2D(380, 350);
-            progressBar2_1.Size2D = new Size2D(140, 4);
-            progressBar2_1.MaxValue = 100;
-            progressBar2_1.MinValue = 0;
-            progressBar2_1.CurrentValue = 30;
-            root.Add(progressBar2_1);
-
-            //progressBar2_2 = new Progress(att);
-            //progressBar2_2.Position2D = new Position2D(80, 560);
-            //progressBar2_2.Size2D = new Size2D(140, 4);
-            //progressBar2_2.MaxValue = 100;
-            //progressBar2_2.MinValue = 0;
-            //progressBar2_2.CurrentValue = 75;
-            //progressBar2_2.UpdateValue();
-            //progressBar2_2.Direction = Progress.DirectionType.Horizontal;
-            //root.Add(progressBar2_2);
+            progressParent = new View()
+            {
+                Position = new Position(480, 200),
+                Size = new Size(900, 630)
+            };
+            progressParent.Layout = new LinearLayout()
+            {
+                LinearOrientation = LinearLayout.Orientation.Horizontal
+            };
+            parent.Add(progressParent);
 
-            board.UpFocusableView = button1;
-            window.Add(root);
-            FocusManager.Instance.SetCurrentFocusView(button1);
+            propParent = new View()
+            {
+                Position = new Position(480, 200),
+                Size = new Size(450, 630)
+            };
+            propParent.Layout = new LinearLayout()
+            {
+                LinearOrientation = LinearLayout.Orientation.Vertical,
+                LinearAlignment = LinearLayout.Alignment.CenterHorizontal,
+                CellPadding = new Size2D(50, 50)
```

Comment:
If you add only one child in layout view and want to move child using padding, please just use padding or set margin to child.

Reply:
Fixed